reference/docs-conceptual/whats-new/differences-from-windows-powershell.md

```diff
@@ -382,6 +399,124 @@ PS> SimpleTest @hash "MyPath"
 Name: Hello; Path: MyPath; Args: -Blah: World
 ```
 
+## Language changes
+
+### Null-coalescing operator `??`
+
+The null-coalescing operator `??` returns the value of its left-hand operand if
+it isn't null. Otherwise, it evaluates the right-hand operand and returns its
+result. The `??` operator doesn't evaluate its right-hand operand if the
+left-hand operand evaluates to non-null.
+
+```powershell
+$x = $null
+$x ?? 100
+```
+
+```Output
+100
+```
+
+In the following example, the right-hand operand won't be evaluated.
+
+```powershell
+[string] $todaysDate = '1/10/2020'
+$todaysDate ?? (Get-Date).ToShortDateString()
+```
+
+```Output
+1/10/2020
+```
+
+### Null-coalescing assignment operator `??=`
+
+The null-coalescing assignment operator `??=` assigns the value of its
+right-hand operand to its left-hand operand only if the left-hand operand
+evaluates to null. The `??=` operator doesn't evaluate its right-hand operand
+if the left-hand operand evaluates to non-null.
+
+```powershell
+$x = $null
+$x ??= 100
+$x
+```
+
+```Output
+100
+```
+
+In the following example, the right-hand operand won't be evaluated.
+
+```powershell
+[string] $todaysDate = '1/10/2020'
+$todaysDate ??= (Get-Date).ToShortDateString()
+```
+
+```Output
+1/10/2020
+```
+
+### Null-conditional operators `?.` and `?[]`
+
+> [!NOTE]
+> This feature was moved from experimental to mainstream in PowerShell 7.1.
+
+A null-conditional operator applies a member access, `?.`, or element access,
+`?[]`, operation to its operand only if that operand evaluates to non-null;
+otherwise, it returns null.
+
+Since PowerShell allows `?` to be part of the variable name, formal
+specification of the variable name is required for using these operators. So it
+is required to use `{}` around the variable names like `${a}` or when `?` is
+part of the variable name `${a?}`.
+
+In the following example, the value of **PropName** is returned.
+
+```powershell
+$a = @{ PropName = 100 }
+${a}?.PropName
+```
+
+```Output
+100
+```
+
+The following example will return null, without trying to access the member
+name **PropName**.
+
+```powershell
+$a = $null
+${a}?.PropName
+```
+
+Similarly, the value of the element will be returned.
+
+```powershell
+$a = 1..10
+${a}?[0]
+```
+
+```Output
+1
+```
+
+And when the operand is null, the element isn't accessed and null is returned.
+
+```PowerShell
+$a = $null
+${a}?[0]
+```
+
+> [!NOTE]
+> Since PowerShell allows `?` to be part of the variable name, formal
+> specification of the variable name is required for using these operators. So
+> it is required to use `{}` around the variable names like `${a}` or when `?`
+> is part of the variable name `${a?}`.
+>
+> The variable name syntax of `${<name>}` should not be confused with the `$()`
+> subexpression operator. For more information, see Variable name section of
+> [about_Variables][about_Variables].
```

Comment:
This seems very nearly a duplicate of the text above the example code, except for the final paragraph in the alert - should we have only one of these?

Reply:
Will address